ssg/constants.py

```diff
@@ -203,7 +203,7 @@
         "cpe:/a:redhat:openstack:13.0",
     ],
     "rhv4": [
-        "cpe:/o:redhat:virtualization:4",
+        "cpe:/a:redhat:enterprise_virtualization_manager:4",
```

Comment:
@yuumasato please also add `cpe:/o:redhat:enterprise_linux:7::hypervisor`

Reply:
@redhatrises And shoud I keep `cpe:/o:redhat:virtualization:4`?

Reply:
@yuumasato although I like it better, no.

Reply:
RHEL7 hypervisor CPE added.